libs/api/books/src/lib/reading-list.service.ts

```diff
@@ -23,6 +23,15 @@ export class ReadingListService {
     });
   }
 
+  async markFinished(id:string): Promise<void> {
+    this.storage.update(list => {
+      return list.map(obj => {
+        if(obj.bookId == id){obj = {...obj,finished:true,finishedDate:new Date().toISOString()}}
```

Comment:
Why is this so condensed? It's not very readable. :( Maybe you could run Prettier?